android/app/src/main/kotlin/org/getlantern/lantern/model/LanternSessionManager.kt

```diff
@@ -324,14 +323,16 @@ class LanternSessionManager(application: Application) : SessionManager(applicati
         setExpired(user.isExpired)
         setIsProUser(user.isProUser)
 
-        val devices = Vpn.Devices.newBuilder().addAllDevices(
-            user.devices.map {
-                Vpn.Device.newBuilder().setId(it.id)
-                    .setName(it.name).setCreated(it.created).build()
-            },
-        ).build()
-        db.mutate { tx ->
-            tx.put(DEVICES, devices)
+        if (user.devices.isNotEmpty()) {
```

Comment:
Just in case you know. better to check for null.

Reply:
Thanks for adding this.